docs/services/blockstorage.rst

```diff
@@ -1,89 +1,187 @@
 .. _block_storage:
 
-Block Storage
+=============
+Block storage
 =============
 
-Commands for Rackspace Cloud Block Storage.
+This section lists all block storage commands supported by Rackspace CLI.
 
-Synopsis
+Commands
 --------
 
-::
+All block storage commands are based on this syntax::
 
    rack block-storage <resource> <action> [command flags]
 
-Commands
---------
+*Command flags* allow you to customize certain attributes of the command,
+such as ``--name`` to name a volume. Type ``rack block-storage <resourse> <action> --help``
```

Comment:
Instead of `resourse` [sic], we've decided to use `subservice`. This is my fault because I originally used `resource` in several places. If you see `resource` mentioned anywhere in the third position of the command, feel free to file an issue or change it yourself.